### ISSUE TITLE
Add graph support directly into search.py

### ISSUE THREAD

**Issue by jmtsuji** [LeeBergstrand/BackBLAST_Reciprocal_BLAST#43]
# Background
Currently, `search.py` sources code from `Graph.py` in the same folder.

# Problem description
Sourcing `Graph.py` like this causes some challenges for the use of `search.py`. Namely, it appears that **`search.py` and `Graph.py` must be in the same folder for `search.py` to easily run**. Although it is possible to specify relatives paths for sourcing code in Python, [this seems to not be the recommended course of action](https://stackoverflow.com/a/16985066) when the files are not part of a proper Python package structure.

# Proposed solution
`Graph.py` is short. Could the code just be integrated into `search.py`? This would also be a good excuse to re-write this tutorial code.

@LeeBergstrand Let me know if you think this is the best course of action.

**Comment by LeeBergstrand** [comment]
@jmtsuji Ya I got that code from some tutorial back in the day. However, that tutorial code is not under licence so we can't actually use it. It should be replaced, either with our own code or with and existing library. For example, the commonly used graph library in Python is NetworkX

https://github.com/networkx/networkx

**Comment by LeeBergstrand** [comment]
![Screen Shot 2019-08-31 at 1 48 00 PM](https://user-images.githubusercontent.com/5819462/64069024-18348800-cbf6-11e9-9978-09daebf9aba4.png)

Does this configuration work? It's okay if they're in the same folder.

**Comment by jmtsuji** [comment]
Yes, the current configuration works. We can use this configuration for
now. However, let's keep this issue open and re-write the graph code (e.g.,
via NetworkX) longer-term.

**Comment by jmtsuji** [comment]
Addressed by #70. `Graph.py` is now replaced by networkx.
